yarn.lock

```diff
@@ -1187,6 +1187,17 @@
     "@jest/types" "^25.2.6"
     jest-mock "^25.2.6"
 
+"@jest/fake-timers@^25.1.0":
+  version "25.5.0"
+  resolved "https://registry.yarnpkg.com/@jest/fake-timers/-/fake-timers-25.5.0.tgz#46352e00533c024c90c2bc2ad9f2959f7f114185"
+  integrity sha512-9y2+uGnESw/oyOI3eww9yaxdZyHq7XvprfP/eeoCsjqKYts2yRlsHS/SgjPDV8FyMfn2nbMy8YzUk6nyvdLOpQ==
+  dependencies:
+    "@jest/types" "^25.5.0"
+    jest-message-util "^25.5.0"
+    jest-mock "^25.5.0"
+    jest-util "^25.5.0"
+    lolex "^5.0.0"
+
 "@jest/fake-timers@^25.2.6":
```

Comment:
dunno if you're up for it, but could be [deduped](https://github.com/atlassian/yarn-deduplicate) with the above

Reply:
Done, also did jest-types and jest-util